NinchatSDKSwift/Implementations/Helpers/SiteConfiguration+Extension.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  String+SiteConfiguration.swift
+//  NinchatSDKSwift
+//
+//  Created by Hassan Shahbazi on 5.12.2019.
+//  Copyright © 2019 Hassan Shahbazi. All rights reserved.
```

Comment:
Btw. the copyright is held by Somia Reality Oy :smile:

Reply:
Use a header like this in the source files:
```
// Copyright (c) 2020 Somia Reality Oy. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.
```
The year should be the file creation year.

Reply:
Agree, but the comments are provided by the Xcode automatically. I have to find a way to change the default style to the one you mentioned.  Otherwise, I have to replace them all once the SDK development is finished.